src/app/cakes/cake.class.ts

```diff
@@ -0,0 +1,30 @@
+export const enum CakeStatus {
```

Comment:
Since we are using a Single File Responsibility file distribution, this Enumerator should live in a separate file, maybe named cake-status.enum.ts.